<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_information</class>
 <widget class="QMainWindow" name="user_information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>246</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>帐号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="search_id_edit"/>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>等级</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="email_edit"/>
     </item>
     <item row="11" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item row="10" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="modify_but">
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="insert_but">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_but">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="search_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="search_but">
         <property name="text">
          <string>用户名搜索</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="12" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>帐号</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_but">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="phone_edit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="id_edit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>手机</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>邮箱</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="key_edit"/>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="search_id_but">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="level_edit"/>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="user_table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>511</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>帐号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>用户名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>邮箱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>手机</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>等级</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
